src/actions/index.js

```diff
@@ -39,6 +40,13 @@ export * from './console'
 export * from './userMessages'
 export * from './disks'
 
+export function appConfigured () {
+  return {
+    type: APP_CONFIGURED,
+    payload: {},
```

Comment:
From time to time if payload is empty we just don't write it (like in https://github.com/oVirt/ovirt-web-ui/blob/master/src/actions/storageDomains.js). I like that style more than writing unnecessary payload.

Reply:
`storageDomains.js` contains the only 2 actions in the app that don't have a payload.  And for the sagas attached to those actions, they push a payload into the action, or define their own, before calling the external action.  I'll leave the `payload: {}` here.

Reply:
Yes, but still this unnecessary string, so why not to start refactor code from here, and remove it completely in the future? Even in sagas examples they don't use `payload: {}` where it not needed. https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/src/actions/index.js

Reply:
Ok, then file a separate issue.  I'm not going to make any further changes to the actions in this PR.